pandas/tests/series/test_rank.py

```diff
@@ -376,3 +376,96 @@ def test_rank_modify_inplace(self):
         s.rank()
         result = s
         assert_series_equal(result, expected)
+
+
+# GH15630, pct should be on 100% basis when method='dense'
+
+@pytest.mark.parametrize('dtype', ['O', 'f8', 'i8'])
```

Comment:
might consider putting these in a single parameterize as well (though not as clear cut as the frame case where you can share the input frame)

Reply:
Yeah...I can't think of a clean way to do this at the moment.  I'll address the other changes first, and then maybe we'll see if any inspiration strikes.  Otherwise, I would leave it alone.